scalyr_agent/builtin_monitors/windows_system_metrics.py

```diff
@@ -102,8 +104,10 @@ def _gather_metric(method, attribute=None, transform=None):
     def gather_metric():
         """Dynamically Generated """
         try:
+            print(attribute)
```

Comment:
I assume that was a debugging change and is not needed anymore :)

Reply:
Woops.